files/build_templates/sonic_debian_extension.j2

```diff
@@ -254,6 +254,7 @@ sudo cp $IMAGE_CONFIGS/hostname/hostname-config.sh $FILESYSTEM_ROOT/usr/bin/
 sudo cp $IMAGE_CONFIGS/misc/docker-wait-any $FILESYSTEM_ROOT/usr/bin/
 
 # Copy updategraph script and service file
+j2 files/build_templates/init_cfg.json.j2 | sudo tee $FILESYSTEM_ROOT/etc/sonic/init_cfg.json
```

Comment:
Move this line down to replace the section you deleted and add a comment like,

```
# Generate initial SONiC configuration file
```

Reply:
Fixed.